### PR TITLE
update page and navigation fix

### LINKED ISSUE #64: Update story page
- [x] to fix header
- [ ] fix navigation links
- [ ] fix conflict

### PR DESCRIPTION
**Pull request by mrxshinji**
Fix #64 

- fixed navigation links (Chart and Recent Activities)
- added hover animation to Link

### CODE REVIEW

File: src/components/header/header.jsx [hunk 1]
```diff
@@ -48,7 +48,7 @@ const Header = () => {
           </div>
           <div className="navbarIconContainer  d-flex justify-content-center align-items-center">
             <a href="ss">
-              <ProfilePic className="profile d-flex justify-content-center align-items-center" />
+              {/* <ProfilePic className="profile d-flex justify-content-center align-items-center" /> */}
```
Comment (mrxshinji) [inline src/components/header/header.jsx:51]: @abdulmaleekadams Can you check on your svg for this profile pic? it is the cause for the blue thing on top right of the page

Comment (abdulmaleekadams) [reply src/components/header/header.jsx:51]: @mrxshinji on it

Comment (mrxshinji) [reply src/components/header/header.jsx:51]: @abdulmaleekadams when you free, review this pr first and approve if its ok. Will merge this first before other pr to create conflict and slowly solve from there i guess

Comment (aftabrehan) [reply src/components/header/header.jsx:51]: @mrxshinji @abdulmaleekadams what's the status of this PR?

Comment (aftabrehan) [reply src/components/header/header.jsx:51]: I think this is resolved in [UPDATE PAGE FIX](https://github.com/appwex/react-ui-components/pull/66), @mrxshinji please confirm it.

Comment (mrxshinji) [reply src/components/header/header.jsx:51]: i have other fix as well, due to some merge conflict wasnt solve properly..

Comment (aftabrehan) [reply src/components/header/header.jsx:51]: Okay

Comment (abdulmaleekadams) [reply src/components/header/header.jsx]: @aftabrehan  I have fixed it

Comment (abdulmaleekadams) [reply src/components/header/header.jsx]: ok I will check

File: src/routes/story-page/story-page.css [hunk 2]
```diff
@@ -5,3 +5,12 @@
...
 }
+
+.story a {
+  transition: all 0.2s;
+  text-decoration: underline;
+}
+
+.story a:hover {
+  transform: scale(1.2);
+}
```
Comment (aftabrehan) [inline src/routes/story-page/story-page.css:16]:
```suggestion
}

```

**Review by aftabrehan — APPROVED** [review]
Excellent PR, @mrxshinji 💪 🚀 💯

**Pull request merged**